### DISCUSSION TITLE
Formal definition of mesh bias parameter

### DISCUSSION THREAD

**Discussion by LukeFisherUKAEA** [idaholab/moose#20584] [Q&A Meshing] [answered]
I have looked for, but have been unable to find, a quantitative definition for how the mesh bias variable behaves.  Is this essentially a geometric series, where the distance between node 0 and _n_ for _n_ > 0 is given by:

_distance_ = sum (_a_ _r_^(_n_ - 1) ) = _a_(1 - _r_^(_n_-1)) / 1 - _r_

where _a_ is the first spacing (i.e. between nodes 0 and 1), _r_ is the bias value and _n_ is the node number?

I suspect there is a very simple explanation for this but I run into problems as there are three parameters to define in the mesh block: the number of nodes, the distance spanned and the bias value.  Does this mean that by choosing the length of a particular dimension and the number of nodes I wish to use, the bias value is fixed?  But if the geometric nature is true then one should also be able to define the first spacing, _a_.   The more I look at it the further away I get from understanding it!  If there is a definition on the MOOSE help pages then I would appreciate a link to it.  Many thanks in advance for the help.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Which mesh generator is this for?
https://mooseframework.inl.gov/source/mesh/GeneratedMesh.html ?

I looked at the code, and the `a` (standard spacing in the non-biased case, not the first spacing anymore) is being modified to respect the number of nodes, the bias value and the distance in that generator. 

Guillaume

**Reply by GiudGiud** [reply]
On a quick note we re getting more mesh biasing work in this PR. Not sure that is relevant for you 
https://github.com/idaholab/moose/pull/20638
